myokit/_sim/cvodessim.py

```diff
@@ -361,6 +361,37 @@ def default_state_sensitivities(self):
             return [list(x) for x in self._s_default_state]
         return None
 
+    def state_sensitivities(self):
```

Comment:
Minor first point: **going forward**, I'm trying to keep all methods sorted alphabetically. It's a very silly thing, but

1. The "natural" order stops making sense if you haven't looked at the code for a while, and isn't the same for all people, so for lack of a better idea alphabetical is now often the norm
2. Existing myokit code often isn't alphabetical, but moving the methods around messes with the git history (makes it look like massive changes happened), so that's why I'm saying "going forward". 
3. Sorry